docs/ensembl-help/using-ensembl/ensembl-apps/species-selector/species-selector-intro.md

```diff
@@ -1,52 +1,127 @@
 
-#  How do I add or remove a species?
+You can use the Species selector to:
 
-The Species selector allows you to view and select the species genomes and assemblies that you wish to view in the Genome browser and Entity viewer.
+- choose a species genome, to work with in other apps and
+- explore the genetic composition of a species genome.
 
-## Selecting a species
+![Species selector showing search box and the 42 most popular species](media/species-selector.png)
 
-In order to start using the Genome browser and Entity viewer, you need to select a species genome and assembly. 
+<figcaption>
 
-There are two ways to select a species genome:
+ The Species selector landing page showing a search box and icons of the 42 most popular species.
 
-1.  Select from an initial selection of 7 popular species genomes. The 42 most popular species genomes will be available in the future.
+</figcaption>
 
-> Select the icon representing your species genome of interest from the 42 most popular species genomes and select the green 'Add' button.
- 
-2.  Search for a species genome
+## How to choose a species genome
 
-> Type the name of a species genome or assembly into the search box and select a species genome from the auto-complete options in the drop-down menu. 
+You must select a species genome to start using any Ensembl app. 
 
-If there are more than one genome assembly available for your species of interest, a drop-down menu will appear allowing you to select your genome assembly of interest. 
-Use the green 'Add' button to select the species genome.
+You can then investigate the genes, variants and orthologues of that species genome in another app ie the Genome browser or Entity viewer.
 
-When a species genome has been selected the species and assembly name will appear at the top of the page.
 
-## Creating a list of species 
+There are two ways to select a species genome in the Species selector:
 
-You can add multiple species genomes and create a list of species genomes in the Species selector. 
+### Choose a genome from the 42 most popular species.
 
-The list of species genomes between them when in other apps such as the Genome browser or Entity viewer.
+<img src="media/popular-species-icons.png" width="320" alt="eight species icons">
 
+<figcaption>
 
-## The Species homepage  
+ Eight icons of the 42 most popular species showing the number of genome assemblies available for each species.
 
-The Species homepage contains more information about the genes, variants of species genome, details about the genome assembly and the controls to manage a list of species genomes. 
+</figcaption>
+
+Select the icon representing your species of interest from the 42 most popular species.
+
+You will see a table of the genome assemblies for that species. 
+
+![A table of genome assembly information](media/genome-assembly-table.png)
+
+Pick the genome assembly you are interested in by marking the box to the left of an assembly and selecting the Add button.
```

Comment:
Here and in other places: is `select` the right verb to describe what one does with a button? Doesn't one _press_ a button rather than _select_ it?